airbyte-webapp/src/components/BarChart/BarChart.tsx

```diff
@@ -1,11 +1,11 @@
 import React from "react";
-import { CartesianGrid, BarChart as BasicBarChart, ResponsiveContainer, XAxis, YAxis, Bar, Label } from "recharts";
+import { Bar, BarChart as BasicBarChart, CartesianGrid, Label, ResponsiveContainer, XAxis, YAxis } from "recharts";
 import { barChartColors, theme } from "theme";
 
 type BarChartProps = {
-  data?: {
+  data: {
     name: string;
-    value: number | number[];
+    value: number;
```

Comment:
Is it safe to remove the number array? How come the type was introduced?

Reply:
This component is only used in one place, and the types now match the incoming data.